packages/laconia-event/src/apigateway/parseWebsocket.js

```diff
@@ -0,0 +1,11 @@
+const getBody = event =>
+  Buffer.from(event.body, event.isBase64Encoded ? "base64" : "utf8").toString();
+
+module.exports = event => {
+  const body = getBody(event);
```

Comment:
There's a small module called `tryJsonParse` within `laconia-event/src`, you can use it here.